### PR TITLE
fix: One can comment with empty message (with only whitespaces)

### LINKED ISSUE #7120: One can comment with empty message (with only whitespaces)
## Issue - Bug

Screenshot 2022-08-29 001817

### Acceptance Criteria (optional)
If only whitespaces are present comments should not be submitted

Estimated effort: 1 hour

### PR DESCRIPTION
**Pull request by gitstart**
# Description

Fixes #7120
If only whitespaces are present comments should not be submitted

## Demo
https://www.loom.com/share/af02cc7f2c394c6da262c8cc573fb2cb

## Testing scenarios

[Please list all the testing scenarios a reviewer has to check before approving the PR]

- [ ] Scenario A
  - Step 1
  - Step 2...

- [ ] Scenario B
  - Step 1
  - Step 2....

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by gitstart** [comment]
@Dschoordsch this is up for review

**Comment by Dschoordsch** [comment]
@igorlesnenko can you please take a look?

File: packages/client/modules/meeting/helpers/ensureHasText.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const ensureHasText = (value: string) => value.trim().length
```
Comment (87prashant) [inline packages/client/modules/meeting/helpers/ensureHasText.ts:1]: I think we could define this function in the same file as the function is very small, a little bit of redundancy is fine I guess.

**Comment by gitstart** [comment]
Hi @87prashant, your review comments have been attended too.

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**